strawberry/contrib/django/tests/test_converter.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+from django.db import models
+
+import strawberry
+from strawberry.contrib.django.converter import convert_django_field
+
+
+def test_converter_returns_correct_type():
+
+    for FieldType in [
```

Comment:
I think it might be better to use pytest fixtures for this :)

and I'd one test for each type (str, int, bool, etc)

Reply:
sorry I meant parametrize http://doc.pytest.org/en/latest/example/parametrize.html

Reply:
Oh cool - didn't know this was a thing! Will update.